### ISSUE TITLE
Issue with installing CellChat package in R studio

### ISSUE THREAD

**Issue by Navya156** [sqjin/CellChat#739]
**I am unable to install Cell Chat package in R studio, even with BiocManager, Github repositories or devtools..

I am getting the following error. Please help me through this. Please guide me to install CellChat package either in OS or windows. Thank you in advance.**

The downloaded source packages are in
	‘/private/var/folders/27/6n67zh0s6bjfkj4j582p7k2r0000gn/T/RtmpIESEZ8/downloaded_packages’
Warning messages:
1: package ‘CellChat’ is not available for Bioconductor version '3.16'

A version of this package for your version of R might be available elsewhere,
see the ideas at
https://cran.r-project.org/doc/manuals/r-patched/R-admin.html#Installing-packages 
2: In install.packages(update[instlib == l, "Package"], l, repos = repos,  :
  installation of package ‘igraph’ had non-zero exit status
> devtools::install_github("jinworks/CellChat")
Downloading GitHub repo jinworks/CellChat@HEAD
These packages have more recent versions available.
It is recommended to update all of them.
Which would you like to update?

1: All                           
2: CRAN packages only            
3: None                          
4: igraph (1.5.1 -> 1.6.0) [CRAN]

Enter one or more numbers, or an empty line to skip updates: 
── R CMD build ───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
✔  checking for file ‘/private/var/folders/27/6n67zh0s6bjfkj4j582p7k2r0000gn/T/RtmpIESEZ8/remotes155d830d8dac7/jinworks-CellChat-39e0f14/DESCRIPTION’ ...
─  preparing ‘CellChat’:
✔  checking DESCRIPTION meta-information ...
─  cleaning src
─  checking for LF line-endings in source and make files and shell scripts
─  checking for empty or unneeded directories
─  building ‘CellChat_2.1.1.tar.gz’
   
ERROR: dependencies ‘ComplexHeatmap’, ‘presto’ are not available for package ‘CellChat’
* removing ‘/Library/Frameworks/R.framework/Versions/4.2/Resources/library/CellChat’
Warning message:
In i.p(...) :
  installation of package ‘/var/folders/27/6n67zh0s6bjfkj4j582p7k2r0000gn/T//RtmpIESEZ8/file155d872824324/CellChat_2.1.1.tar.gz’ had non-zero exit status
> install.packages("ComplexHeatmap")
Warning in install.packages :
  package ‘ComplexHeatmap’ is not available for this version of R

**Comment by sqjin** [comment]
@Navya156 Obviously, you should check how to install  dependencies ‘ComplexHeatmap’, ‘presto’
